MeshLib could not be built in the ubuntu24 container

I tried to build MeshLib in the provided environment, and it didn't work.

## Steps to reproduce

1. `git clone --recurse-submodules https://github.com/MeshInspector/MeshLib.git`
2. `docker build . -f docker/ubuntu24Dockerfile -t meshlib`
3. `docker run --rm -it meshlib` and then `cd ~/ && git clone --recurse-submodules https://github.com/MeshInspector/MeshLib.git && cd MeshLib`
4. The following step produces an error:
```shell
user@c0b13035bfd1:~/MeshLib$ ./scripts/build_source.sh
Project build script started.
You could find output in /home/user/MeshLib/build_source_22-07-2024_19:01:50.log
Build with emscripten? Press (y) in 5 seconds to build (y/s/N) (s - singlethreaded)
Emscripten OFF, singlethread 0
Build MeshLib Release? Press (n) in 5 seconds to cancel (Y/n)
Release ON
Build MeshLib Debug? Press (y) in 5 seconds to build (y/N)
....
CMake Warning (dev) at source/MRCuda/CMakeLists.txt:16 (find_package):
  Policy CMP0146 is not set: The FindCUDA module is removed.  Run "cmake
  --help-policy CMP0146" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at /usr/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find CUDA: Found unsuitable version ".", but required is at least
  "12" (found /usr/local/cuda/)
Call Stack (most recent call first):
  /usr/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:598 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.28/Modules/FindCUDA.cmake:1291 (find_package_handle_standard_args)
  source/MRCuda/CMakeLists.txt:16 (find_package)
```
5. Even if I try to turn the CUDA off, it still doesn't work, building with the following command gives the same error
```shell
user@c0b13035bfd1:~/MeshLib$ export MR_CMAKE_OPTIONS="-DCMAKE_POLICY_DEFAULT_CMP0146=OLD -DMESHINSPECTOR_CUDA_SUPPORT=OFF"
```

--- a/docker/readme.md
+++ b/docker/readme.md
@@ -1,34 +1,15 @@
-### Some useful commands
+### MeshLib docker images
 
-Build image with
-```
-$ docker build -f ./docker/fedora39Dockerfile -t meshlib/fedora39-build-server .
-$ docker build -f ./docker/ubuntu22Dockerfile -t meshlib/ubuntu22-build-server .
-```
+The Dockerfiles in this directory (and the corresponding `meshlib/meshlib-*` images on [Docker Hub](https://hub.docker.com/u/meshlib)) are designed for MeshLib CI/CD only: they provide the build environments for the GitHub Actions workflows and are not intended for local development.
 
-Run a temporary container:
+In particular, the images contain prebuilt thirdparty libraries in `/usr/local/lib/` (`meshlib-thirdparty-lib/` in the Linux images; `emscripten/`, `emscripten-single/` and `emscripten-wasm64/` in the emscripten ones) — outside the source tree, where the build does not find them by itself. If you still want to build MeshLib from source in a container, first link them into the repository root, as the CI workflows do:
 ```
-$ docker run --rm -it meshlib/fedora39-build-server bash
-```
-Run a container in background:
-```
-$ docker run -d --name angry_fedora meshlib/fedora39-build-server tail -f /dev/null
-```
-[Start an existing container](https://docs.docker.com/engine/reference/commandline/container_start/) and "Attach STDOUT/STDERR and forward signals" (-a), "Attach container's STDIN" (-i):
-```
-$ docker container start -ai angry_fedora
-```
-Attach to a running (!) container as root:
-```
-$ docker exec -u 0 -it angry_fedora bash
-```
-Show all containers:
-```
-$ docker ps -a
-```
-Delete all containers:
-```
-$ docker rm -f $(docker ps -a -q)
+$ ln -s /usr/local/lib/meshlib-thirdparty-lib/lib ./lib
+$ ln -s /usr/local/lib/meshlib-thirdparty-lib/include ./include
+$ ln -s /usr/local/lib/meshlib-thirdparty-lib/share ./share
 ```
 
-
+Build an image locally:
+```
+$ docker build -f ./docker/ubuntu24Dockerfile -t meshlib/meshlib-ubuntu24 .
+```

